package.json

```diff
@@ -103,14 +105,14 @@
 		"react": "17.0.2 - latest",
 		"react-dom": "17.0.2 - latest",
 		"uuid": "9.0.0 - latest",
-		"vega": "5.22.1 - latest",
-		"vega-embed": "6.21.0 - latest",
-		"vega-lite": "5.6.0 - latest"
+		"vega-embed": "6.23.0 - latest"
```

Comment:
Should we add `vega-embed` to the peerDependencies to use that module if already installed?

Reply:
It shouldn't already be installed by the user because most people shouldn't already be doing vega charts on their own.
I'm kinda following how `vega/react-vega` does it https://github.com/vega/react-vega/blob/master/packages/react-vega/package.json